docs: add link to a proper aws guide for configuring credentials (#4091)
aws/copilot-cli: site/content/docs/credentials.en.md

> AWS Copilot uses AWS credentials to access AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate application's workloads.

Reviewer: ```suggestion
AWS Copilot uses AWS credentials to access the AWS API, store and look up an [application's metadata](concepts/applications.en.md), and deploy and operate an application's workloads.
```

> You can learn more on how to configure AWS credentials at [Quick configuration with `aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).

Reviewer: This is super helpful for first-time users. I remember being confused by this when I first started! Thank you.

> You can learn more on how to configure AWS credentials at [Quick configuration with `aws configure`](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).

Reviewer 2: ```suggestion
You can learn more on how to configure AWS credentials in the [AWS CLI's documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
```
